### PR TITLE
use asyncio.current_task in python 3.7

### LINKED ISSUE #23: Python 3.7: PendingDeprecationWarning: Task.current_task() is deprecated, use asyncio.current_task() instead
I get a lot of deprecation warning spam in my unit tests, e.g.:
```
/home/olli/.pyenv/versions/3.7.0/envs/ml-microservice-3.7.0/lib/python3.7/site-packages/aiotask_context/__init__.py:57: PendingDeprecationWarning: Task.current_task() is deprecated, use asyncio.current_task() instead
  context = asyncio.Task.current_task(loop=loop).context
```

### PR DESCRIPTION
**Pull request by diefans**
this resolves #23

Also
- randomized tests
- fixed dependent tests
- updated ci requirements
- added tox.ini

### CODE REVIEW

**Comment by diefans** [comment]
travis should be ok now. I do not know why it is still failed, when the new python 3.7 run was successful???

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,12 @@ python:
   - "3.5"
   - "3.6"
 
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
```
[the remark this answers is not in the document]
Comment (diefans) [reply .travis.yml]: travis has issues with installing python 3.7. See the broken builds for 3.,7

[the remark this answers is not in the document]
Comment (diefans) [reply .travis.yml]: I also think tox makes sense, when testing locally - I am curious how you do this without.
Later this day I will propose the travis changes...

**Comment by argaen** [comment]
merged, I'll try to find a moment to release the new version :)

**Comment by diefans** [comment]
very nice! Thanks...

**Comment by argaen** [comment]
Thank you for the contribution!

**Comment by argaen** [comment]
@diefans I've just released version 0.6.0

**Pull request merged**